### ISSUE TITLE
Allow using "aria-readonly" on elements with role=combobox

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#798]
```aria-readonly```  has been added to supported states and properties in ARIA 1.1 spec for role ```combobox```  https://www.w3.org/TR/wai-aria-1.1/#combobox

**Comment by WilcoFiers** [comment]
This doesn't seem like a thing that should be used unless there is proper support for it in AT. At least in VoiceOver it doesn't seem to be supported yet, so I don't think this is something axe-core should allow just yet.

**Comment by iamrafan** [comment]
Ah! ok. I missed out on testing it with Voiceover

**Comment by ferBonnin** [comment]
@iamrafan revisiting this issue since we received a user report on this. Can you please test this with voiceOver again?

**Comment by iamrafan** [comment]
@ferBonnin I re-tested, it is still not supported in VoiceOver
